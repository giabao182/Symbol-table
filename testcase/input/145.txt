INSERT s string
ASSIGN s 'it\'s complicated'