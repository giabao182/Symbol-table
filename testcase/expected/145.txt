success
TypeMismatch: ASSIGN s 'it\'s complicated'